misc/twist_mux/launch/twist_mux.launch

```diff
@@ -2,7 +2,7 @@
   <arg name="joy_vel_in"  default="input_joy/cmd_vel"/>
   <arg name="joy_vel_out" default="joy_vel"/>
 
-  <arg name="cmd_vel_out" default="twist_mux/cmd_vel"/>
```

Comment:
Please do not modify a 3rd party launch file, the correct way to resolve this issue is to call `roslaunch twist_mux.launch` with `<rosparam name="cmd_vel_out" value="cmd_vel"/>`